huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_l7rule_test.go

```diff
@@ -76,33 +76,33 @@ func testAccCheckElbV3L7RuleDestroy(s *terraform.State) error {
 		}
 
 		if l7policyID == "" {
-			return fmtp.Errorf("Unable to find l7policy_id")
+			return fmt.Errorf("unable to find l7policy_id")
 		}
 
-		_, err := l7rules.GetRule(lbClient, l7policyID, rs.Primary.ID).Extract()
+		_, err := l7policies.GetRule(elbClient, l7policyID, rs.Primary.ID).Extract()
 		if err == nil {
-			return fmtp.Errorf("L7 Rule still exists: %s", rs.Primary.ID)
+			return fmt.Errorf("the L7 Rule still exists: %s", rs.Primary.ID)
 		}
 	}
 
 	return nil
 }
 
-func testAccCheckElbV3L7RuleExists(n string, l7rule *l7rules.Rule) resource.TestCheckFunc {
+func testAccCheckElbV3L7RuleExists(n string, l7rule *l7policies.Rule) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		rs, ok := s.RootModule().Resources[n]
 		if !ok {
-			return fmtp.Errorf("Not found: %s", n)
+			return fmt.Errorf("not found: %s", n)
 		}
 
 		if rs.Primary.ID == "" {
-			return fmtp.Errorf("No ID is set")
+			return fmt.Errorf("no ID is set")
 		}
 
-		config := acceptance.TestAccProvider.Meta().(*config.Config)
-		lbClient, err := config.ElbV3Client(acceptance.HW_REGION_NAME)
+		cfg := acceptance.TestAccProvider.Meta().(*config.Config)
+		lbClient, err := cfg.ElbV3Client(acceptance.HW_REGION_NAME)
```

Comment:
please use "elbClient', consistent with the previous

Reply:
done